### DISCUSSION TITLE
Disable Auto Theme Toggle

### DISCUSSION THREAD

**Discussion by kevinzakka** [pradyunsg/furo#499] [Questions] [answered]
Is there a way to disable the auto theme and just have a toggle between light and dark?

**Comment by pradyunsg** [comment]
No. See #24 for the details.

**Reply by pradyunsg** [reply]
See also https://github.com/pradyunsg/furo/issues/28

**Reply by kevinzakka** [reply]
Hi @pradyunsg, maybe I am misunderstanding the above linked issues but I don't specifically want to disable light/dark. I just don't want that auto toggle. It would be nice if the browser would directly assign either black or white and there's a single toggle for the user to force change to whatever they prefer. Does that make sense?

**Reply by pradyunsg** [reply]
Ah, right. You want to have light mode by default, ignoring the user's declared browser preference?

**Reply by kevinzakka** [reply]
Can we not just go directly to whatever the browser asks for without an 3 state toggle?

i.e. if my browser is in dark mode, it automatically goes to dark mode and there is just 1 button to go to light at the top and vice versa if my browser is in light mode, there is just 1 button to go to dark mode at the top.

**Reply by pradyunsg** [reply]
I think I understand what you want now.

**Comment by pradyunsg** [comment]
I mean, I understand now what you want. What I don’t understand is why you want that. Could you elaborate on this? How is the auto setting problematic?

**Reply by kevinzakka** [reply]
It's just a very minor nit I had and I was looking to see if there was an easy way to tweak it. I just find it slightly unintuitive to click on a toggle once and not have anything happen, it might throw off first time users who don't know what the "auto" one does. If it can't be done at the moment then no worries at all, was just asking :)

**Reply by pradyunsg** [reply]
> I just find it slightly unintuitive to click on a toggle once and not have anything happen

That's not how stuff works -- the first click *always* changes the theme. If it's not doing that for you, I'd like a reproducer of the behaviour you're seeing. :)

**Comment by pradyunsg** [comment, marked as answer]
I'm still waiting on a reproducer for the behaviour that you're seeing.

**Reply by kevinzakka** [reply]
Hi @pradyunsg, apologies for the late reply. Please disregard my bug issue, the first click does indeed change the theme. Appreciate this library, thanks for everything!
